project/npda/migrations/0006_alter_npdauser_email.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1 on 2024-08-21 12:48
+
+import citext.fields
+from django.db import migrations
+from django.contrib.postgres.operations import CITextExtension
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "npda",
+            "0005_rename_organisation_name_paediatricdiabetesunit_lead_organisation_name_and_more",
+        ),
+    ]
+
+    operations = [
+        CITextExtension(),
```

Comment:
This will probably fail during CI as only the main admin user can create extensions in Azure Flexible Postgres. So I'll run `CREATE EXTENSION citext` manually on the database before deploying.